streams/examples/main/java/org/apache/kafka/streams/examples/pageview/PageViewTypedJob.java

```diff
@@ -30,22 +31,35 @@
 import org.apache.kafka.streams.kstream.KStream;
 import org.apache.kafka.streams.kstream.KTable;
 import org.apache.kafka.streams.kstream.KeyValueMapper;
+import org.apache.kafka.streams.kstream.ValueJoiner;
 import org.apache.kafka.streams.kstream.Windowed;
 import org.apache.kafka.streams.StreamsConfig;
 
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Properties;
 
+/**
+ * Implements the "PageView Stats" program that joins the pageview stream with
+ * the user profile table to count the number of pageview per user region.
+ * Assumes the input stream is named "streams-pageview-input".
+ *
+ * This example shows how to:
+ *   1. write stateful computations like joins / aggregations using the high-level KStream DSL.
+ *   2. use specific data types (e.g. JSON POJO, Avro Specific Binding, etc) for serdes in Kafka Streams.
```

Comment:
I'd clarify to sth like:

> 2. use specific data types (here: JSON pojo;  but can also be Avro specific bindings, etc.) for serdes in Kafka Streams.

To make it clear that this example does not showcase Avro usage.